app/assets/stylesheets/base.scss

```diff
@@ -274,6 +274,11 @@ table {
 .fact-col {
   max-width: 500px;
   white-space: normal;
+  font-family: monospace;
```

Comment:
From the Jira task:
> The font used for displaying numbers will be changed to a monospaced font.

Is this enough that `monospace` font will be inherited from user's system or do we have in Patternfly something special?

> The new font will be consistent across all relevant parts of the application.

Right now it's just for facts, common parameters and ansible. What about others, do we have a list of places that needs to be changed in follow up tasks?

> The font size will remain the same as the current font for numbers.

We have font for numbers? Never heard of.

cc @theforeman/ui-ux